docs/understanding-airbyte/database-data-catalog.md

```diff
@@ -0,0 +1,72 @@
+* `stream_reset`
+  * Each record in this table represents a stream in a connection that is enqueued to be reset. It can be thought of as a queue. Once the stream is reset, the record is removed from the table.
+* `operation`
+  * The `operation` table transformations for a connection beyond the raw output produced by the destination. The two options are: `normalization`, which outputs Airbyte's basic normalization. The second is `dbt`, which allows a user to configure their own custom dbt transformation. A connection can have multiple operations (e.g. it can do `normalization` and `dbt`).
+  * If the `operation` is `dbt`, then the `operator_dbt` column will be populated with a JSON blob with the schema from [OperatorDbt](airbyte-config/config-models/src/main/resources/types/OperatorDbt.yaml).
+  * If the `operation` is `normalization`, then the `operator_dbt` column will be populated with a JSON blob with the scehma from [OperatorNormalization](airbyte-config/config-models/src/main/resources/types/OperatorNormalization.yaml).
+  * Operations are scoped by workspace, using the `workspace_id` column.
+* `connection_operation`
+  * This table joins the `operation` table to the `connection` for which it is configured. 
+* `workspace_service_account`
+  * I don't know what this is. Should we drop it?
+* `actor_oauth_parameter`
+  * The name of this table is misleading. It refers to parameters to be used for any instance of an `actor_definition` (not an `actor`) within a given workspace. For OAuth, the model is that a user is provisioning access to their data to a third party tool (in this case the Airbyte Platform). Each record represents information (e.g. client id, client secret) for that third party that is getting access. 
+  * These parameters can be scoped by workspace. If `workspace_id` is not present, then the scope of the parameters is to the whole deployment of the platform (e.g. all workspaces).
+  * The `actor_type` column tells us whether the record represents a Source or a Destination.
+  * The `configuration` column is a JSON blob. The schema of this JSON blob matches the schema specified in the `spec` column in the `advanced_auth` field of the JSON blob. Keep in mind this schema is specific to each connector (e.g. the schema of Hubspot and Salesforce are different), which is why this column has to be a JSON blob.
+* `secrets`
+  * This table is used to store secrets in open-source versions of the platform that have not set some other secrets store. This table allows us to use the same code path for secrets handling regardless of whether an external secrets store is set or not. This table is used by default for the open-source product.
+* `airbyte_configs_migrations` is metadata table used by Flyway (our database migration tool). It is not used for any application use cases.
+* `airbyte_configs`
```

Comment:
@sherifnada or @davinchia do you remember what this table was for?

Reply:
I don't think it's used anymore. @benmoriceau or @subodh1810 do you remember why this wasn't removed?

Reply:
I think this is where we kept everything before @subodh1810 did the work to [normalise tables](https://github.com/airbytehq/airbyte/issues/7867).

The latest created_at/modified_at timestamps in prod is early Jan, so I think we can drop this now.

<img width="1745" alt="Screenshot 2022-09-09 at 9 43 41 AM" src="https://user-images.githubusercontent.com/9772859/189400091-f8777a1a-c583-460d-9d0e-daa060606398.png">

Reply:
Should it be kept to keep the history of the configs before a the normalization?

Reply:
okay. so clarification:
* `airbyte_configs` in the config db is empty (so there's nothing to save here anyway).
* `airbyte_configs` in the cloud db still has some old data (not inclined to think it is useful at this point).

Given that, I'm inclined to drop both.